examples/diskwipe/main.go

```diff
@@ -3,19 +3,21 @@ package main
 import (
 	"context"
 	"fmt"
+	"time"
 
 	"github.com/metal-toolbox/ironlib/actions"
 	"github.com/sirupsen/logrus"
 )
 
-// This example invokes ironlib and wipes the disk /dev/sdZZZ
+// This example invokes ironlib and wipes the disk /dev/sdZZZ with a timeout of 1 day
 
 func main() {
 	logger := logrus.New()
 	logger.Formatter = new(logrus.JSONFormatter)
 	logger.SetLevel(logrus.TraceLevel)
 	sca := actions.NewStorageControllerAction(logger)
-	ctx := context.Background()
+	ctx, cancel := context.WithTimeout(context.Background(), 86400*time.Second)
```

Comment:
could have been `24*time.Hour` but meh not a big deal especially if its in example code

Reply:
Would it be worthwhile to conduct a test to validate context cancellation, even though the purpose of the context is already evident from the messages? Just clarifying, I've chosen 86400 seconds to potentially force it to 1 and trigger the context cancellation due to timeout, it could have been 24*time.Hour, but as you said it's not a significant issue inside example code.